Source/RapyutaSimulationPlugins/Private/Tools/LimitRTFFixedSizeCustomTimeStep.cpp

```diff
@@ -0,0 +1,144 @@
+// Copyright Epic Games, Inc. All Rights Reserved.
+
+#include "Tools/LimitRTFFixedSizeCustomTimeStep.h"
+
+#include "Misc/App.h"
+#include "Misc/ConfigCacheIni.h"
+
+ULimitRTFFixedSizeCustomTimeStep::ULimitRTFFixedSizeCustomTimeStep(const FObjectInitializer& ObjectInitializer)
+    : Super(ObjectInitializer), StepSize(0.01), TargetRTF(1)
+{
+    float frameRate = 100.0;
+    if (GConfig->GetFloat(TEXT("/Script/Engine.Engine"), TEXT("FixedFrameRate"), frameRate, GEngineIni))
+    {
+        StepSize = 1.0 / frameRate;
+    }
+    FApp::SetUseFixedTimeStep(true);
+    FApp::SetFixedDeltaTime(StepSize);
+
+    GConfig->GetFloat(TEXT("/Script/Engine.Engine"), TEXT("TargetRTF"), TargetRTF, GEngineIni);
+    UE_LOG(LogRapyutaCore,
+           Display,
+           TEXT("[ULimitRTFFixedSizeCustomTimeStep][ULimitRTFFixedSizeCustomTimeStep]: StepSize: %f, TargetRTFL %f"),
+           StepSize,
+           TargetRTF);
+
+    LastPlatformTime = FPlatformTime::Seconds();
+}
+
+bool ULimitRTFFixedSizeCustomTimeStep::Initialize(UEngine* InEngine)
+{
+    return true;
+}
+
+void ULimitRTFFixedSizeCustomTimeStep::Shutdown(UEngine* InEngine)
+{
+    // Empty but implemented because it is PURE_VIRTUAL
+}
+
+bool ULimitRTFFixedSizeCustomTimeStep::UpdateTimeStep(UEngine* InEngine)
+{
+    UpdateApplicationLastTime();    // Copies "CurrentPlatformTime" (used during the previous frame) in "LastTime"
+    WaitForSync();
+    return false;    // false means that the Engine's TimeStep should NOT be performed.

```

Comment:
```suggestion
    // Copies "CurrentPlatformTime" (used during the previous frame) in "LastTime"
    UpdateApplicationLastTime();
    WaitForSync();
    // false means that the Engine's TimeStep should NOT be performed.
    return false;
```
comment convention, to be before the snippet, mostly to avoid being broken by clang-format